src/resources/books.md

```diff
@@ -1,14 +1,14 @@
 ---
 layout: default
 permalink: /resources/books
-title: Books about Dart
+title: Books and Videos about Dart
 description: Read all about it! Here's a collection of books about Dart.
 toc: false
 ---
 
-Here's a collection of books about Dart.
+Here's a collection of books and videos about Dart.
```

Comment:
Should we qualify the "videos" part somehow? We're not including our videos (although maybe we should point to them from here). And I assume we won't include short videos. We're really talking about major resources here. I'm not sure what words to use, though...